test/parallel/test-https-truncate.js

```diff
@@ -14,7 +15,7 @@ const key = fs.readFileSync(common.fixturesDir + '/keys/agent1-key.pem');
 const cert = fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem');
 
 // number of bytes discovered empirically to trigger the bug
-const data = Buffer.allocUnsafe(1024 * 32 + 1);
+const data = Buffer.alloc(1024 * 32 + 1);
```

Comment:
> May I ask why alloc is preferred over allocUnsafe in a test?

I don't feel strongly about it, but here's the thinking:

* If we use `Buffer.alloc()`, then we get a buffer with unpredictable contents. While the contents don't play a role in the test here, all things being equal, I prefer having repeatable, predictable tests over tests with unpredictable inputs involved (unless, of course, the purpose is to do something like fuzz testing).

* This is not performance-critical code, so the performance hit of zero-filling is not a concern.

* The very name of `Buffer.allocUnsafe()` seems to suggest "Don't use it unless you need to and know what you're doing." We don't *need* it here, so...

Again, I don't feel particularly strongly about it, but it does seem to me that if we can use either `alloc()` or `allocUnsafe()` without any real difference, then we should default to `alloc()`.